### PR TITLE
Working with CC UI & Feedback fixes

### PR DESCRIPTION
**Pull request by velikovmartin**

### CODE REVIEW

File: controls/radrichtextbox/features/content-controls/content-controls.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ The following content controls are not supported by __Microsoft Word__:
 
 ## Common Content Controls Properties
 
->note Some of the content controls` properties are marked as **(Non-exportable)** which means they live only in the RadRichTextBox model.
+>note Some of the content controls` properties are marked as **(Non-exportable)** which means they are specific only for RadRichTextBox and are not serialized to any of the supported document formats. These properties are included because they are considered useful and could enhance the experience using the content controls in a RadRichTextBox environment.
```
Comment (TanyaDimitrova) [inline controls/radrichtextbox/features/content-controls/content-controls.md:45]: How are they useful? Add some description next to the properties to describe that

File: controls/radrichtextbox/features/content-controls/content-controls.md [hunk 2]
```diff
@@ -81,23 +101,76 @@ The __ComboBox__ and __DropDownList__ provide the user with options to choose fr
     - __DisplayText:__ **(Non-exportable)** This property holds the displayed in the ComboBox/DropdownList text.
     - __Value:__ **(Non-exportable)** This property holds the value, which can be propagated through a data-binding relation.
 
+#### Example 2: Setting ComboBox Properties
```
Comment (TanyaDimitrova) [inline controls/radrichtextbox/features/content-controls/content-controls.md:104]: Use normal casing for the captions of the snippets/tables/figures

File: controls/radrichtextbox/features/content-controls/content-controls.md [hunk 3]
```diff
@@ -81,23 +101,76 @@ The __ComboBox__ and __DropDownList__ provide the user with options to choose fr
...
 ### Date
 
 The __Date__ content control allows you to enter a date by using a calendar. The date content control has the following properties:
 * __DateFormat:__ Allows you to get/set the format string of the date. If it is omitted the default date format for the language is used.
-* __LanguageId:__ Allows you to get/set the CultureInfo object for the date format.
-* __DateTime:__ The current selected date, stored as string. 
+* __Language:__ Allows you to get/set the CultureInfo object for the date format.
+* __FullDate:__ The current selected date, stored as string.
 * __Calendar:__ Allows you to select the calendar type.
+* __DateMappingType:__ Gets or sets the data type that is used for storing mapped date time value.
```
Comment (TanyaDimitrova) [inline controls/radrichtextbox/features/content-controls/content-controls.md:126]: I don't understand that. Try to describe it better, add it to the example below

File: controls/radrichtextbox/features/content-controls/working-with-content-controls-ui.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+title: Working with Content Controls UI
+page_title: Working with Content Controls UI
+description: This article describes how to work with the Content Controls UI
+slug: working-with-content-controls-ui
+tags: content controls, richtextbox
+published: True
+---
+
+# Content Controls UI
```
Comment (TanyaDimitrova) [inline controls/radrichtextbox/features/content-controls/working-with-content-controls-ui.md:10]: Add some short description about the contents of the topic

File: controls/radrichtextbox/features/content-controls/working-with-content-controls-ui.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* **Date Picker** content control
+
+### Content Controls Commands
+The following commands related to the content controls functionality are available in the Commands property of the RadRichTextBox:
+
+* **InsertStructuredDocumentTagCommand**
+
+    In order to successfully execute the InsertStructuredDocumentTagCommand you have to pass the wanted content control type (**SdtType**) as a CommandParameter:
+
+    #### [XAML] Example 2: Binding the command
```
Comment (TanyaDimitrova) [inline controls/radrichtextbox/features/content-controls/working-with-content-controls-ui.md:34]: Example 1

File: controls/radrichtextbox/features/content-controls/working-with-content-controls-ui.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+                        SmallImage="{telerik:IconResource IconRelativePath=16/contentControl-picture.png,IconSources={StaticResource IconSources}}" 
+                        telerik:RadRichTextBoxRibbonUI.RichTextCommand="{Binding InsertStructuredDocumentTagCommand}"
+                        CommandParameter="Picture"
+                        Size="Small" telerik:ScreenTip.Title="Picture Content Control">
+        </telerik:RadRibbonButton>
+    {{endregion}}
+
+* **ShowContentControlPropertiesDialogCommand**
+
+    #### [XAML] Example 1: Binding the command
```
Comment (TanyaDimitrova) [inline controls/radrichtextbox/features/content-controls/working-with-content-controls-ui.md:48]: Example 2

File: controls/radrichtextbox/features/content-controls/content-controls.md (hunk 1 above)
Comment (velikovmartin) [reply controls/radrichtextbox/features/content-controls/content-controls.md:45]: I think these properties are understandably described.

**Pull request merged**